modules/virt-importing-vm-cli.adoc

```diff
@@ -76,7 +76,7 @@ EOF
 ----
 <1> Specify the RHV logical network and vNIC profile.
 <2> Specify the {VirtProductName} network.
-<3> If `storageMappings` are specified in both the ResourceMapping and the VirtualMachineImport CRs, the VirtualMachineImport CR takes precedence.
+<3> If `storageMappings` are specified in both the ResourceMapping and the virtual machine import CRs, the virtual machine import CR takes precedence.
```

Comment:
Same as above -- anywhere we see CR or custom resource or Custom Resource should get the backticks/pascalcase